tests/unittests/odm_tests.py

```
@@ -109,3 +109,85 @@ def test_get_indexed_values(self):
+        data = {
+            "field1": "value1",
+            "unknown_field": {
+                "_cls": "UnknownDocumentType",
+                "data": "some_data",
+            },
+        }
+
+        result = foou.sanitize_unknown_embedded_docs(data)
+
+        # The unknown document should be converted to DynamicEmbeddedDocument
+        self.assertEqual(result["field1"], "value1")
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)